src/screens/Home/Home.tsx

```diff
@@ -1,12 +1,48 @@
 import type { FC } from "react"
+import type { User } from "@react-native-google-signin/google-signin"
+
+import { useEffect, useState } from "react"
 import { useNavigation } from "@react-navigation/native"
 import { StyleSheet, Image } from "react-native"
+import {
+  GoogleSignin,
+  GoogleSigninButton,
+} from "@react-native-google-signin/google-signin"
 import { Box, Press, Typography } from "../../components"
 
 const assetsUrl = process.env.PMO_ASSETS
 
 const Home: FC = () => {
   const navigation = useNavigation()
+  const [userInfo, setUserInfo] = useState<User | null>()
+
+  async function handleSignIn() {
+    try {
+      const userInfo = await GoogleSignin.signIn()
+      setUserInfo(userInfo)
+    } catch (error) {
+      setUserInfo(null)
+      console.log("GoogleSignin.signIn() error", error)
```

Comment:
`console.warn()` will pop up on screen in the emulator without needing the dev tools open. I'm wondering if thats a better dev experience. Obviously neither is a good solution for a production app. Go ahead and merge this, just putting it out there for your opinion.

Reply:
I think this should be a `console.error` if we keep it.

This doesn’t have any loading or error state, but it should, right? If an error occurs, we’d want to display that to the user saying they couldn’t sign in, or out, and ideally show any error code/info that we can.